components/gitpod-protocol/go/gitpod-config-types.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2020 Gitpod GmbH. All rights reserved.
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
```

Comment:
This file auto-generated. I added git hook which regenerates it on commit, see [.pre-commit-config.yaml](https://github.com/gitpod-io/gitpod/pull/11653/files#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9)